src/cart/crt_internal_types.h

```diff
@@ -14,7 +14,7 @@
 #define CRT_CONTEXT_NULL         (NULL)
 
 #ifndef CRT_SRV_CONTEXT_NUM
-#define CRT_SRV_CONTEXT_NUM (64)	/* Maximum number of contexts */
+#define CRT_SRV_CONTEXT_NUM (128) /* Maximum number of contexts */
```

Comment:
as a note this will increase a client side memory footprint. Clients contain 2 cache tables - uri and hg addr; each at most contains  num_ranks * CRT_SRV_CONTEXT_NUM entries. CRT_SRV_CONTEXT_NUM entries allocated at once (as an array) per each discovered rank. In case of a uri cache, each entry is a string such as (ofi+tcp://127.0.0.1:12345) while hg_addr cache the entry is a mercury handle.

Reply:
yup 128 should be ok, as we used to have 256 limit for a long time. still, if we had any scaled up test that was on the border of oom, this might cause it to go over, so something to be aware of.